sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/rest_client.hpp

```diff
@@ -374,6 +374,7 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
         Nullable<DateTime> SourceIfUnmodifiedSince;
         Nullable<std::string> EncryptionKey;
         Nullable<std::vector<uint8_t>> EncryptionKeySha256;
+        Nullable<std::string> EncryptionAlgorithm;
```

Comment:
We need to test only these 3 operations support CPK, and other operations don't support CPK.